### PR TITLE
Add pagination, Valerie's green buttons, dark purple body test

### PR DESCRIPTION
**Pull request by cmacaulay**
- Adds will_paginate gem so be sure to bundle!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -32,6 +32,7 @@ gem 'jbuilder', '~> 2.5'
 gem 'bcrypt', '~> 3.1.7'
 gem 'unicorn'
 gem 'rubocop', require: false
+gem 'will_paginate'
```
Comment (meyerhoferc) [inline Gemfile:35]: 👍

File: app/controllers/items_controller.rb [hunk 2]
```diff
@@ -19,6 +19,6 @@ def sort_reviews
   end
 
   def jumbotron
-    @items = Item.all
+    @items = Item.paginate(:page => params[:page], :per_page => 30)
```
Comment (meyerhoferc) [inline app/controllers/items_controller.rb:22]: 🥇

File: app/views/admin/orders/index.html.erb [hunk 3]
```diff
@@ -35,7 +36,7 @@
       <option value = 'completed'>Completed</option>
       <option value = 'all'>All Orders</option>
     </select></br>
-    <%= f.submit "filter", class: "waves-effect waves-light btn" %>
+    <%= f.submit "filter", class: "btn" %>
```
Comment (meyerhoferc) [inline app/views/admin/orders/index.html.erb:39]: awesome

File: app/views/shared/_nav.html.erb [hunk 4]
```diff
@@ -28,7 +28,7 @@
 <nav>
    <div class="nav-wrapper">
      <div class ="container">
-       <a href="<%= items_path %>" class="brand-logo">Weasleys' Wizard Wheezes</a>
+       <a href="<%= root_path %>" class="brand-logo">Weasleys' Wizard Wheezes</a>
```
Comment (meyerhoferc) [inline app/views/shared/_nav.html.erb:31]: much more intuitive

File: spec/features/admin/items/admin_can_add_a_new_item_spec.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    fill_in("title") with: item.title
+    fill_in("description") with: item.description
+    fill_in("price") with: item.price
+    fill_in("image_tage") with: item.image_tag
+    fill_in("title") with: category.id )
+
+    click "Submit"
+
+    expect(current_page).to be(items_path)
+    expect(page).to have_content(item.title)
```
Comment (meyerhoferc) [inline spec/features/admin/items/admin_can_add_a_new_item_spec.rb:26]: like this addition

File: spec/features/admin/items/admin_can_edit_an_item_spec.rb [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    fill_in("description") with: item_two.description
+    fill_in("price") with: item_two.price
+    fill_in("image_tage") with: item_two.image_tag
+    fill_in("title") with: category.id )
+
+    click "Submit"
+
+    expect(current_page).to be(items_path)
+    expect(page).to have_content(item_two.title)
+    expect(page).to_not have_content(item_one.title)
```
Comment (meyerhoferc) [inline spec/features/admin/items/admin_can_edit_an_item_spec.rb:31]: 🔥

**Review by meyerhoferc — APPROVED** [review]
I approve, ready for second review & merge @CjMoore @valerietrudell

**Review by CjMoore — COMMENTED** [review]
Looks good. Merging.

**Pull request merged**